### DISCUSSION TITLE
What's allowing SearX to be selectable on Chromium mobile ?

### DISCUSSION THREAD

**Discussion by KaKi87** [searxng/searxng#2133] [Q&A]
Hello,

SearX instances get added to the *Recently visited* section of search engine selection on Chromium mobile :

![Screenshot_20230128-175317_Kiwi Browser](https://user-images.githubusercontent.com/21284089/215278897-18be4cd9-fcf4-4b50-807b-3277e6083012.png)

What exactly makes it work ?

I tried :
- `<form role="search">`
- `<input type="search">`
- `<link rel="search"
type="application/opensearchdescription+xml">`

Without success.

Thanks

**Comment by jonas-w** [comment]
You were on the right track with open search, you will also need to specify the path to the xml file with all the info about the search engine. e.g. https://searx.be/opensearch.xml

More on this here:
https://developer.mozilla.org/en-US/docs/Web/OpenSearch

**Reply by KaKi87** [reply]
Well, of course, I tried this :
https://git.kaki87.net/KaKi87/startpage/src/branch/master/src/index.html#L9
https://git.kaki87.net/KaKi87/startpage/src/branch/master/src/public/opensearch.xml

But it didn't work... And it seems there's no error log whatsoever to figure out what's wrong with that.

Can you help me ?

Thanks

**Reply by return42** [reply]
My search engine is located at https://darmarit.org/searx/ in the intro page of https://darmarit.org/ I added this line, that points to `href="/searx/opensearch.xml"` ..

```html
<link title="searx" type="application/opensearchdescription+xml" rel="search" href="/searx/opensearch.xml">
```

My brand is shown in this commit https://github.com/return42/searxng/commit/21c39000d6 .. you need to set `base_url`.

----

For a instance that is located at https://search.example.org simply set  `base_url: https://search.example.org`
